lib/rules/no-unneeded-ternary.js

```diff
@@ -134,7 +134,17 @@ module.exports = {
                         node,
                         loc: node.consequent.loc.start,
                         message: "Unnecessary use of conditional expression for default assignment.",
-                        fix: fixer => fixer.replaceText(node, `${astUtils.getParenthesisedText(sourceCode, node.test)} || ${astUtils.getParenthesisedText(sourceCode, node.alternate)}`)
+                        fix: fixer => {
```

Comment:
Nitpick: It might be better to use 

```js
const nodeAlternateText = node.alternate.type === "ConditionalExpression"
    ? `(${sourceCode.getText(node.alternate)})`
    : sourceCode.getText(node.alternate);
```

That way, `nodeAlternate` doesn't need to be reassigned several times.